cedar-lean/Cedar/Thm/Validation/Typechecker/UnaryApp.lean

```diff
@@ -122,7 +122,7 @@ theorem type_of_neg_is_sound {x₁ : Expr} {c₁ c₂ : Capabilities} {env : Env
       have ⟨i, h₈⟩ := instance_of_int_is_int h₇
       subst h₈
       simp [apply₁, intOrErr]
-      cases h₉ : i.neg?
+      cases _h₉ : i.neg?
```

Comment:
Curious:  why do we need the underscore before `h9` here?

Reply:
To suppress a warning about it being unused. I'm sure there's some syntax to just omit it, but `_` doesn't work.

Reply:
Got it.

Reply:
`cases i.neg?` should work